kde_plasmoid_debian/Readme.md

```diff
@@ -0,0 +1,66 @@
+# Mycroft Plasmoid // Release 2.0
+#### Mycroft Ai Plasmoid and Skills for KDE Plasma 5 Desktop
+
+Auto Install Instructions:
+
+1. git clone https://github.com/MycroftAI/installers/kde_plasmoid_debian
```

Comment:
will change this to wget instead of git as that is easier, there are parts in the install scripts that shouldn't be run under sudo (plasma skills) so it would be better if the users do it currently via chmod